### ISSUE TITLE
Problem when using Bevy Fox model

### ISSUE THREAD

**Issue by pinkponk** [komadori/bevy_mod_outline#6]
Using the Fox.glb model:
https://github.com/bevyengine/bevy/tree/latest/assets/models/animated

Causes 

```
2022-09-20T21:54:50.979944Z ERROR wgpu::backend::direct: Handling wgpu errors as fatal by default    
thread 'main' panicked at 'wgpu error: Validation Error

Caused by:
    In a RenderPass
      note: encoder = `<CommandBuffer-(0, 2, Vulkan)>`
    In a draw command, indexed:false indirect:false
      note: render pipeline = `outline_stencil_pipeline`
    the pipeline layout, associated with the current render pipeline, contains a bind group layout at index 1 which is incompatible with the bind group layout associated with the bind group at 1

```

Can it have something to do with it being animated? I did try adding: AutoGenerateOutlineNormalsPlugin but no luck

**Comment by komadori** [comment]
The joints in the model seems a likely cause. They are normally handled in the shader using extra vertex attributes, which is not the case for the outline shader.

I've been short of free time recently (apols for the delay in replying), but I will look into this as soon as I'm able.

**Comment by komadori** [comment]
Fixed in 0.2.4.

**Comment by pinkponk** [comment]
Love it thanks!
![image](https://user-images.githubusercontent.com/14301446/195724768-0432eec1-81e9-43cb-98df-5c455888505f.png)
